AWS-docker-compose.yml

```diff
@@ -1,8 +1,7 @@
 version: '2'
 services:
     bugzilla:
-        image: uneet/bugzilla-customisation:$COMMIT
-        restart: on-failure
```

Comment:
@kaihendry why do we need to change this behavior?

Reply:
ecs-cli doesn't understand the restart keyword, so removed it